### PR TITLE
Grammar fix: by it's administrator -> by its administrator

### PR DESCRIPTION
**Pull request by jaller94**
Fixes https://github.com/vector-im/element-web/issues/22017

Submitted as an employee of Element (New Vector Inc.).

element-web notes: none

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/i18n/strings/bg.json [hunk 1]
```diff
@@ -2461,7 +2461,7 @@
     "Spaces": "Пространства",
     "Check your devices": "Проверете устройствата си",
     "%(deviceId)s from %(ip)s": "%(deviceId)s от %(ip)s",
-    "This homeserver has been blocked by it's administrator.": "Този Home сървър е бил блокиран от администратора му.",
+    "This homeserver has been blocked by its administrator.": "Този Home сървър е бил блокиран от администратора му.",
```
Comment (turt2live) [inline src/i18n/strings/bg.json:2464]: we cannot change strings in these files. Only files modified by `yarn i18n` are possible.

**Review by turt2live — CHANGES_REQUESTED** [review]

File: src/i18n/strings/bg.json (hunk 1 above)
Comment (jaller94) [reply src/i18n/strings/bg.json:2464]: I don't understand what files must not be changed and why.

Comment (jaller94) [reply src/i18n/strings/bg.json]: done

File: src/i18n/strings/en_EN.json [hunk 2]
```diff
@@ -805,7 +805,6 @@
     "%(brand)s is experimental on a mobile web browser. For a better experience and the latest features, use our free native app.": "%(brand)s is experimental on a mobile web browser. For a better experience and the latest features, use our free native app.",
     "Use app": "Use app",
     "Your homeserver has exceeded its user limit.": "Your homeserver has exceeded its user limit.",
-    "This homeserver has been blocked by it's administrator.": "This homeserver has been blocked by it's administrator.",
```
Comment (jaller94) [inline src/i18n/strings/en_EN.json:808]: This got "removed" because the correct spelling already existed in another place of the code.

**Review by turt2live — APPROVED** [review]
Thanks!

**Pull request merged**